airbyte-integrations/connectors/source-stripe/CHANGELOG.md

```diff
@@ -2,3 +2,5 @@
 
 ## 0.1.8
 Start using the new Source Acceptance tests
+## 0.1.14
```

Comment:
this file is not needed -- let's remove it and point to the docs in `docs/...`

Reply:
Done.